mock-apps/save-and-return-simulator/app.rb

```diff
@@ -0,0 +1,20 @@
+require 'sinatra'
+require 'json'
+require 'sinatra/json'
+require 'pry'
+
+post '/email' do
+  data = request.body.read
+  parsed_data = JSON.generate(JSON.parse(data))
+  if File.write('tmp/params.json', parsed_data)
+    status 200
+    body "Ok"
+  else
+    status 400
+    body "Unable to generate params"
+  end
+end
+
+get '/email' do
+  json JSON.parse(File.read('tmp/params.json'))
```

Comment:
Should we rescue the missing file error and return a `: not_found`?

Reply:
Yes, definitely! ♻️